subsys/settings/Kconfig

```diff
@@ -104,3 +110,25 @@ config SETTINGS_FS_MAX_LINES
 	depends on SETTINGS && SETTINGS_FS
 	help
 	  Limit how many items stored in a file before compressing
+
+config SETTINGS_NVS_SECTOR_SIZE
+	int "Sector size of the NVS settings area"
+	default 4096
+	depends on SETTINGS && SETTINGS_NVS
+	help
+	  The sector size to use for the NVS settings area in bytes.
+
+config SETTINGS_NVS_SECTOR_COUNT
+	int "Sector count of the NVS settings area"
+	default 3
+	depends on SETTINGS && SETTINGS_NVS
+	help
+	  Number of sectors used for the NVS settings area
+
+config SETTINGS_NVS_OFFSET
+	int "Offset of the NVS settings area"
+	default 0
+	depends on SETTINGS && SETTINGS_NVS
```

Comment:
I would define SETTINGS_NVS_OFFSET to default to CONFIG_FLASH_STORAGE_OFFSET, but I don't know if this is possible.